server/tinylicious/src/resourcesFactory.ts

```diff
@@ -26,7 +26,7 @@ import {
 export class TinyliciousResourcesFactory implements utils.IResourcesFactory<TinyliciousResources> {
     public async create(config: Provider): Promise<TinyliciousResources> {
         // Pull in the default port off the config
-        const port = utils.normalizePort(process.env.PORT || "3000");
+        const port = utils.normalizePort(process.env.PORT || "35843");
```

Comment:
you need to stage this. first make this change, release, and consume in client with the driver change

Reply:
ok, PR changed to only change tinylicious